app/src/community/res/layout/development_activity_login.xml

```diff
@@ -71,6 +71,7 @@
 
                 <com.google.android.material.textfield.TextInputEditText
                     android:id="@+id/community_login_screen_name_edittext"
+                    android:autofillHints="username"
```

Comment:
Should this be a string resource?

Reply:
No this corresponds to a constant that is defined in the Android View class [here](url). Not sure how to reference that constant from the xml though.